test/client/fixtures/sandbox/iframe-test.js

```diff
@@ -43,6 +43,22 @@ test('event should not raise if a cross-domain iframe is appended', function ()
     $iframe.remove();
 });
 
+asyncTest('iframe initialization must be synchronously', function () {
```

Comment:
Check text with @VasilyStrelyaev. Not all iframes should be initialized sync, it should be in title. It should be in regression section with appropriate `GH-{num}` postfix